lib/openfeature/sdk/client.rb

```diff
@@ -1,18 +1,83 @@
 # frozen_string_literal: true
 
+require "concurrent"
```

Comment:
This comes from [concurrent-ruby](https://github.com/ruby-concurrency/concurrent-ruby) right?

I'm not seeing it in the gemspec on main, or in this PR though.

Reply:
Good call out. I've removed it for now